fitters/include/DAF.h

```diff
@@ -89,6 +95,7 @@ class DAF : public AbsKalmanFitter {
    * Also sets #minIterations_ and #maxIterations_.
    */
   void setAnnealingScheme(double bStart, double bFinal, unsigned int nSteps);
+  void setAnnealingScheme(double bStart, double bFinal, unsigned int nSteps, unsigned int minIter, unsigned int  maxIter);
```

Comment:
I suggest to add a docstring explaining the difference between this method and the one above.

Reply:
done